spras/containers.py

```diff
@@ -239,6 +242,40 @@ def run_container_docker(container: str, command: List[str], volumes: List[Tuple
     # finally:
     return out
 
+# TODO any issue with creating a new client each time inside this function?
+def run_container_podman(container: str, command: List[str], volumes: List[Tuple[PurePath, PurePath]], working_dir: str, environment: str = 'SPRAS=True'):
```

Comment:
To test the Podman functionality, a test case should be added to each algorithm test (similar to the singularity test cases).